src/Exceptionless.AppHost/Extensions/ElasticsearchExtensions.cs

```diff
@@ -100,7 +99,7 @@ public static IResourceBuilder<ElasticsearchResource> WithDataVolume(this IResou
     {
         ArgumentNullException.ThrowIfNull(builder);
 
-        return builder.WithVolume(name ?? VolumeNameGenerator.CreateVolumeName(builder, "data"), "/usr/share/elasticsearch/data");
+        return builder.WithVolume(name ?? VolumeNameGenerator.Generate(builder, "data"), "/usr/share/elasticsearch/data");
```

Comment:
https://github.com/dotnet/aspire/issues/7788

Reply:
Yep, need to remove our version of VolumeNameGenerator